.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Necessary ?

Reply:
Yeap' it helps really ... for vscode users 😜